spring-cloud-stream-schema/src/main/java/org/springframework/cloud/stream/schema/avro/AvroMessageConverterAutoConfiguration.java

```diff
@@ -61,6 +66,12 @@ public AvroSchemaRegistryClientMessageConverter avroSchemaMessageConverter(
 					this.avroMessageConverterProperties.getSchemaLocations());
 		}
 		avroSchemaRegistryClientMessageConverter.setPrefix(this.avroMessageConverterProperties.getPrefix());
+		avroSchemaRegistryClientMessageConverter.setCacheManager(cacheManager());
 		return avroSchemaRegistryClientMessageConverter;
 	}
+
+	@Bean
```

Comment:
Use `ConditionalOnMissingBean` or else this will effectively replace any already defined cache manager.